modules/creating-oci-resources-services.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="creating-oci-resources-services_{context}"]
+= Creating OCI resources and services
+
+Create {oci-first} resources and services so that you can establish infrastructure with governance standards that meets your organization requirements.
+
```

Comment:
@faermanj . Does the DNS / zone OCI configuration step slot in here? What is the key objective of this step? 

On the creating a stack step in another section/module, zone_dns exists, but this is more of a case of specifying an already created DNS zone?